src/uiHelpers.h

```diff
@@ -130,6 +129,9 @@ void uiCallback_reject(ui_callback_t* cb);
 void assert_uiPaginatedText_magic();
 void assert_uiPrompt_magic();
 
+void validate_uiPaginatedText_canFitStringIntoHeader(const char* str);
+void validate_uiPaginatedText_canFitStringIntoFullText(const char* str);
```

Comment:
I'd expect these would be used within `ui_displayPaginatedText` (is it true that everything we display is displayed through this function?). Also, I am slightly in favor of having these as boolean functions (so that anyone can do with their results whatever he wants, e.g. use them in an assertion or validation).